lib/validates_timeliness/railtie.rb

```diff
@@ -2,7 +2,13 @@ module ValidatesTimeliness
   class Railtie < Rails::Railtie
     initializer "validates_timeliness.initialize_active_record", :after => 'active_record.initialize_timezone' do
       ActiveSupport.on_load(:active_record) do
-        ValidatesTimeliness.default_timezone = ActiveRecord::Base.default_timezone
+        default_timezone = if ActiveRecord::VERSION::MAJOR >= 7
```

Comment:
Hi, I think that the `respond_to?` approach used in #218 may be a better approach for this conditional check